docs/sdk/serverless-signing/broadcast-and-collect.md

```diff
@@ -0,0 +1,42 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Broadcast and Collect Within an Action
+
+## Overview
+
+Within a Lit action, you may wish to run a specific operation on all nodes within the network and collect the responses for processing as a set of data from within the node's trusted execution enviorment (TEE). This is helpful if you wish to perform operations over the data as a set. Where all values are grouped and returned to every other node from within the lit action context.
+
+# Broadcasting and Collecting a fetch response
+
+```js
+const code = `
+const url = "https://api.weather.gov/gridpoints/TOP/31,80/forecast";
+const resp = await fetch(url).then((response) => response.json());
+const temp = resp.properties.periods[0].temperature;
+
+const temperatures = await Lit.Actions.broadcastAndCollect({
+  name: "temperature",
+  value: temp,
+});
+
+// at this point, temperatures is an array of all the values that all the nodes got
+const median = temperatures.sort()[Math.floor(temperatures.length / 2)];
+Lit.Actions.setResponse({response: median});
+`;
+
+const client = new LitNodeClient({
+    litNetwork: 'cayenne',
+});
+await client.connect();
+const res = client.executeJs({
+    code,
+    sessionSigs: {} // your session
+    jsParams: {
+      publicKey: "<your pkp public key>",
+      sigName: 'fooSig',
+    }
```

Comment:
Can we add this line here (in `jsParams`) as suggested by you in all the five .md files:
> // or add any other value to send to lit action

Reply:
yes please

Reply:
I actually think it is better in this example to not add this comment as the example Lit Action does not use any other options and is to just show how  use the new lit action functions. Since its an advanced guide i think it wouldnt make sense